packages/headless/src/controllers/commerce/core/facets/generator/headless-commerce-facet-generator.test.ts

```diff
@@ -61,100 +63,97 @@ describe('FacetGenerator', () => {
     initCommerceFacetGenerator();
```

Comment:
I see that initCommerceFacetGenerator() was ran twice before so I understand why you removed the beforeEach() here, but what is the benefit of not grouping the tests with describe() as they were before?

Reply:
There's no advantage really, I decided to flatten because everything we're testing about this controller happens upon initialization 😅